src/Apps/Invoice/Routes/__tests__/InvoiceDetailRoute.jest.tsx

```diff
@@ -68,10 +68,7 @@ describe("InvoiceDetailRoute", () => {
   it("renders invoice details correctly", () => {
     expect(
       screen.getByTestId("invoice-name").textContent
-    ).toMatchInlineSnapshot(`"Name: John Doe"`)
-    expect(
-      screen.getByTestId("invoice-email").textContent
-    ).toMatchInlineSnapshot(`"Email: john@example.com"`)
+    ).toMatchInlineSnapshot(`"John Doe <john@example.com>"`)
```

Comment:
This actually reminded me that my main question/'issue' around this super simple app was the testing. I added these just as a basic 'info properly rendered', however `getByText` was proving problematic due to use of `<em>` and the text spanning multiple elements.

I even went down a little rabbit hole of writing a custom matcher/regex, before just doing a snapshot (which I don't love).

Do you recommend against snapshots (I feel like they're easily overlooked plus are perfectly brittle), or was it maybe a smell that I was finding it hard to right vanilla text matchers (I shouldn't be using `<em>` like that and so forth)?

Reply:
Yah, 100% avoid snapshots. 

```
expect(screen.getByText("Foo")).toBeInTheDocument()
```
just keep it simple

Reply:
But to answer the q, to fix that problem, you do a regex:

```
screen.getByText(/foo bar baz/)
```
that'll cross over line breaks, etc.

Reply:
(Removed snapshots in here: https://github.com/artsy/force/pull/14733)